build-review-apps.md

```diff
@@ -15,7 +15,7 @@ which constitutes the **"Review App"**.
   - [Setup](#setup)
   - [Workflow](#workflow)
   - [Create script](#create-script)
-  - [You're done! 🎉](#youre-done-)
+  - [You're done! 🎉](#your-done-)
```

Comment:
Everything's awesome except this change. This will break the table of contents. May you revert this line? 😄

Thanks!

Reply:
thanks! will update 🔨

Reply:
done 🔨